docs/README.md

```diff
@@ -162,6 +162,11 @@ Don't use it as a coordinating conjunction; start a new sentence instead.
 For example, this is an example.
 ([#738](https://github.com/web-platform-dx/web-features/pull/738#discussion_r1537762579), [#742](https://github.com/web-platform-dx/web-features/pull/742))
 
+##### indicates
```

Comment:
It looks like this was motivated by https://github.com/web-platform-dx/web-features/pull/1436. If it's only popped up once I don't think we need to document it.

Note that it's used in a way allowed by this guideline here:

https://github.com/web-platform-dx/web-features/blob/ee7fe7940c1e25b649236c12bbb2b3349aa49ea7/features/modulepreload.yml#L2

Reply:
OK, I've removed "indicates" from this PR.